repo_sync/git_log.py

```diff
@@ -0,0 +1,21 @@
+"""
+Git_log
+
+Copyright (C) 2021 Netpro Project RepoSync
+"""
+
+import click
+
+
+@click.command()
+@click.option('--repo-git', type=click.File('r'), prompt=True, help="view log", required=True)
+def git_log(repo_git: str) -> None:
+    """
+    repo_git (str): read log.txt
+    """
+    with repo_git as f:
```

Comment:
ファイルは指定しません。仮に指定してもRust側で開くのでPythonでは開きません。

というか、ファイル開くのは

```suggestion
    with open(repo_git) as f:
```

では？